### DISCUSSION TITLE
No Ability to search for folders

### DISCUSSION THREAD

**Discussion by dig12345** [standardnotes/forum#2738] [Suggestions]
There is currently no way to find a folder easily without browsing the whole tree to find. Either tags should be included in search results or there should be another method to search through the folder list to find the folder you're looking for but don't remember exactly where it is in the tree.

**Comment by JaspalSuri** [comment]
Hi @dig12345, I can confirm that this currently isn't possible and that it's something that we'd like to add at some point in the future. 🙂

**Comment by dig12345** [comment]
@JaspalSuri Any update on this? With all the changes to folders recently I thought this would've been included..

**Comment by JaspalSuri** [comment]
I’m no longer affiliated with Standard Notes, but I think this might be more of a long-term feature.

**Comment by dig12345** [comment]
Can we get an official response on this? Finding a folder seems to be a pretty basic functionality. Either an additional search box at the bottom of the folders pane? Or a toggle filter on the current search box. Something to find a folder... This slows down my productivity with SN...
